ChangeLog

```diff
@@ -1,5 +1,34 @@
 OpenContainers Specifications
 
+Changes with v1.2.1:
+
+	Additions:
+
+	* zos updates (#1273)
+	* Add support for windows CPU affinity (#1258)
+	* specs-go: sync SCMP_ARCH_* constants with libseccomp main (#1229)
+	* Add CPU affinity to executed processes (#1253)
```

Comment:
This should also mention #1261 (which is a followup to #1253)

Reply:
IOW

```suggestion
	* Add CPU affinity to executed processes (#1253, #1261)
```

Reply:
#1261 has been marked as its own change in line 25 - are you suggesting I combine them under this PR for changeLog?

Reply:
I will move 1261 beside this PR